tools/pot/tests/utils/data_helper.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from copy import deepcopy
+
+import numpy as np
+
+
+class NumpyEncoder(json.JSONEncoder):
+    """ Special json encoder for numpy types """
+    # pylint: disable=W0221, E0202
+
+    def default(self, o):
+        if isinstance(o, np.integer):
+            return int(o)
+        if isinstance(o, np.floating):
+            return float(o)
+        if isinstance(o, np.ndarray):
+            return o.tolist()
+        return json.JSONEncoder.default(self, o)
+
+
+def dump_intermediate_data(local_path, data):
+    data = json.dumps(deepcopy(data), cls=NumpyEncoder)
+    local_file = open(local_path, 'w')
+    json.dump(data, local_file)
+
+def get_ref_stats(stats_path):
```

Comment:
I think it should be given a more generic name like `load_json` and use the function in other tests.

https://github.com/openvinotoolkit/openvino/blob/473bded2b2c37908196d5cd08afba3bafd0dae2b/tools/pot/tests/test_config.py#L27-L28
https://github.com/openvinotoolkit/openvino/blob/473bded2b2c37908196d5cd08afba3bafd0dae2b/tools/pot/tests/test_config.py#L87-L90
https://github.com/openvinotoolkit/openvino/blob/473bded2b2c37908196d5cd08afba3bafd0dae2b/tools/pot/tests/test_statistics_collector.py#L63-L64
https://github.com/openvinotoolkit/openvino/blob/473bded2b2c37908196d5cd08afba3bafd0dae2b/tools/pot/tests/test_telemetry.py#L47-L48
https://github.com/openvinotoolkit/openvino/blob/473bded2b2c37908196d5cd08afba3bafd0dae2b/tools/pot/tests/test_unify_scales.py#L72-L74